components/org.wso2.carbon.identity.data.publisher.audit.common/src/main/java/org/wso2/carbon/identity/data/publisher/audit/common/AuditDataPublisherUtils.java

```diff
@@ -56,4 +68,22 @@
         }
         return metaData;
     }
+
+    /**
+     * Get a string of comma separated values from an array
+     *
+     * @param values The values array which needs to be converted
+     * @return The string of comma separated values
+     */
+    public static String getCommaSeparatedList(String[] values) {
```

Comment:
Add a new line after the method signature.

Reply:
Added.